BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -215,8 +216,25 @@ class CbrainTask::<%= name %> < <%= descriptor['cbrain:inherits-from-class'] ||
       %>
 %     end
     }
+%   end
 
+%   unless seps.empty?
+    # Input command-line separators (with flags) used with keys in command-line
+    # substitution.
+    seps = {
+%     key_width = max_width.(seps, 'command-line-key')  + "''".length
+%     seps.each do |sep|
+      <%=
+        "%-#{key_width}s => %s" % [
```

Comment:
I'm talking about argument that are user-supplied  using the form... e.g. a text input box. What is someone enters, as a value for one of these options, `ahaha"; rm -rf /`   (with a single double quote in the middle to trigger the injection)